contracts/MuonNodeManager.sol

```diff
@@ -265,6 +274,45 @@ contract MuonNodeManager is
         }
     }
 
+    /**
+     * @dev Returns a list of nodes that have been edited.
+     * @param _lastEditTime The time of the last edit.
+     * @param index The index to start retrieving the edited nodes or zero.
+     * @return nodesList An array of edited nodes.
+     * @return lastIndex The index of the last retrieved edit log in the `editLogs` array.
+     */
+    function getEditedNodes(uint256 _lastEditTime, uint256 index)
+        public
+        view
+        returns (Node[] memory nodesList, uint256 lastIndex)
+    {
+        uint256 startIndex = index == 0 ? editLogs.length - 1 : index - 1;
+        nodesList = new Node[](100);
```

Comment:
We can get a `_max` as a parameter and use `nodesList = new Node(_max)` instead of hard coding 100